aoc2022/src/main/scala/day10.scala

```diff
@@ -17,38 +23,57 @@ object day10 extends App:
   private val start1: Long =
     System.currentTimeMillis
 
-  private val input: List[String] =
+  case class Instruction(ex: String, value: Int)
```

Comment:
You can make `Instruction` a sealed trait with two implementations: `Noop` and `Addx`